### PR TITLE
[FR] translation chap2 done

### PR DESCRIPTION
**Pull request by mgemard**
Finished chapter 2

### CODE REVIEW

File: wasm.yaml [hunk 1]
```diff
@@ -999,6 +1054,34 @@ pages:
...
+        N'hésite pas non plus à regarder s'il n'existe pas de librairies Rust pour te
+        simplifier la tâche.
+        Par exemple, [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) est une
+        option populaire.
+
+        Dans le prochain chapitre, nous parlerons d'une technique pour appeler
+        dynamiquement les fonctions JavaScript et nous aborderons
+        le ramasse-miettes (garbage collector)!
+    fr_c:
+      title: Chapter 2 Conclusion
```
Comment (VincentFoulon80) [inline wasm.yaml:1073]:
```suggestion
      title: Chapitre 2 - Conclusion
```

File: wasm.yaml [hunk 2]
```diff
@@ -1011,6 +1094,11 @@ pages:
       title: Chapter 3 - Dynamic Invocation
       content_markdown: |
         In this chapter we will talk about how we can create our own dynamic functions at runtime to call from WebAssembly.
+    fr:
+      title: Chapter 3 - Invocation dynamique
```
Comment (VincentFoulon80) [inline wasm.yaml:1098]:
```suggestion
      title: Chapitre 3 - Invocation dynamique
```

File: wasm.yaml [hunk 3]
```diff
@@ -1011,6 +1094,11 @@ pages:
...
+      content_markdown: |
+        Dans ce chapitre, nous expliquerons comment créer nos propres fonctions
+        dynamiques (créées lors de l'exécution) et être appelées depuis WebAssembly.
```
Comment (VincentFoulon80) [inline wasm.yaml:1101]:
```suggestion
        dynamiques (créées lors de l'exécution) et les appeler depuis WebAssembly.
```

**Comment by mgemard** [comment]
I have added your changes.

Thanks for the review!

**Comment by richardanaya** [comment]
Awesome work you two! :) :handshake:

**Pull request merged**